### PR TITLE
Fix #16486 - conf-python-3-dev package.

### LINKED ISSUE #16486: Fix conf-python3-dev for CentOS and OpenSUSE
From https://github.com/ocaml/opam-repository/pull/15687
```
#=== ERROR while compiling conf-python-3-dev.1 ================================#
# context              2.0.5 | linux/x86_64 | ocaml-base-compiler.4.09.0 | git+file:///home/opam/opam-repository
# path                 ~/.opam/4.09/.opam-switch/build/conf-python-3-dev.1
# command              /usr/bin/make
# exit-code            2
# env-file             ~/.opam/log/conf-python-3-dev-20-f9e6ac.env
# output-file          ~/.opam/log/conf-python-3-dev-20-f9e6ac.out
### output ###
# cc `python3-config --includes` -o test test.c
# /bin/sh: python3-config: command not found
# test.c:1:10: fatal error: Python.h: No such file or directory
#  #include "Python.h"
#           ^~~~~~~~~~
# compilation terminated.
# make: *** [Makefile:2: build] Error 1
```
Reopening because issue is still there, https://github.com/ocaml/opam-repository/issues/15688 was closed by the bot even though I made a comment.

### PR DESCRIPTION
**Pull request by XVilka**

### CODE REVIEW

**Comment by XVilka** [comment]
Problem is very strange:
```
[default] synchronised from file:///home/opam/opam-repository
[ERROR] At /home/opam/.opam/repo/default/packages/conf-python-3-dev/conf-python-3-dev.1/opam:20:26:
        '.' is not a valid token
[ERROR] Strict mode: aborting
[ERROR] Could not update repository "default": OpamStd.OpamSys.Exit(30)
```

File: packages/conf-python-3-dev/conf-python-3-dev.1/opam [hunk 1]
```diff
@@ -8,16 +8,16 @@ build: [
   [make]
 ]
 depexts: [
-  ["python3-dev"] {os-family = "debian"}
+  ["python3-dev"] {os-family = "debian" | os-family = "suse"}
   ["python3-devel"] {os-distribution = "fedora"}
-  ["python3-dev"] {os-distribution = "alpine"}
+  ["python3-devel" "epel-release"] {os-distribution = "centos"}
+  ["python3-dev"] {"os-distribution = "alpine"}
```
Comment (rjbou) [inline packages/conf-python-3-dev/conf-python-3-dev.1/opam:14]:
```suggestion
  ["python3-dev"] {os-distribution = "alpine"}
```
trailing quote :)

File: packages/conf-python-3-dev/conf-python-3-dev.1/opam [hunk 2]
```diff
@@ -10,14 +10,15 @@ build: [
 depexts: [
   ["python3-dev"] {os-family = "debian"}
   ["python3-devel"] {os-distribution = "fedora"}
+  ["python3-devel" "epel-release"] {os-distribution = "centos"}
+  ["python3-devel"] {os-family = "suse"}
   ["python3-dev"] {os-distribution = "alpine"}
   ["python3"] {os = "macos" & os-distribution = "homebrew"}
```
Comment (mseri) [inline packages/conf-python-3-dev/conf-python-3-dev.1/opam:16]: Can we add macports?
```suggestion
  ["python3"] {os = "macos" & os-distribution = "homebrew"}
  ["python38"] {os = "macos" & os-distribution = "macports"}
```

Comment (mseri) [reply packages/conf-python-3-dev/conf-python-3-dev.1/opam:16]: Done :P

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**